data_steward/test/unit_test/validation_test.py

```diff
@@ -78,7 +78,7 @@ def test_errors_csv(self, mock_check_cron):
             actual_result = test_util.read_cloud_file(self.hpo_bucket, folder_prefix + common.ERRORS_CSV)
             with open(test_util.BAD_PERSON_FILE_BQ_LOAD_ERRORS_CSV, 'r') as f:
                 expected = f.read()
-                self.assertEqual(expected, actual_result)
+                self.assertTrue(set(expected).issubset(set(actual_result)))
```

Comment:
@nishanthpp93 this converts `expected` and `actual_result` from strings into sets of `char` and compares those sets; probably not the condition we want to test. How about checking there is a row where `type=error` and `file_name=person.csv`?

Reply:
Yes that works. I've pushed the changes.